src/github.ts

```diff
@@ -139,14 +139,12 @@ export function loadIssueByTerm(term: string) {
     }
     return response.json();
   }).then(results => {
-    if (results.total_count === 0) {
-      return null;
-    }
-    if (results.total_count > 1) {
-      // tslint:disable-next-line:no-console
-      console.warn(`Multiple issues match "${q}". Using earliest created.`);
+    for (const item of results.items) {
+      if (item.title === term) {
```

Comment:
Would changing this line to use indexOf still fix your issue? I'm concerned about the impact this change will have because the [config documentation](https://utteranc.es/#heading-mapping) say "issue title **contains** term" rather than "**equals** term".

```diff
- if (item.title === term) {
+ if (item.title.toLowerCase().indexOf(term.toLowerCase()) !== -1) {
```

Reply:
I supposed that this could also work, but I'm not sure if there aren't potential edge cases. I'll apply your suggestion regardless, because it will fix the initial issue.